packages/client/components/assignment-selections/index.tsx

```diff
@@ -0,0 +1,92 @@
+import React from 'react'
+import {
+  ExpansionPanel,
+  ExpansionPanelSummary,
+  ExpansionPanelDetails,
+  Button
+} from '@material-ui/core'
+import ExpandMoreIcon from '@material-ui/icons/ExpandMore'
+import styled from 'styled-components'
+
+const DetailContainer = styled.div`
+  display: flex;
+  width: 100%;
+  flex-direction: column;
+  justify-content: space-between;
+`
+
+const ButtonContent = styled.div`
+  display: flex;
+  flex-direction: column;
+  width: 100%;
+  align-items: flex-start;
+`
+
+const TitleText = styled.div`
+  font-size: 20px;
+`
+
+const StyledButton = styled(Button)`
+  && {
+    text-transform: none;
+  }
+`
+
+interface AssignmentButtonProps {
+  main: string
```

Comment:
I'm unsure what `main` refers to, can you think of another variable name, or put a comment above the property explaining what it means/does?

Reply:
Might be better to just name it `title` and pass in the entire title. (eg. <... title={'Review Given'}/>)